index.html

```diff
@@ -16,7 +16,7 @@
   </header>
   <body>
     <section id="synopsis">
-      <img src="assets/skull2.gif" />
+      <img src="assets/skull2.gif" alt="spinning skull" />
```

Comment:
I noticed that you did add an alt tag to this image but some of your other wonderful images didn't use an alt tag. It's a super easy thing to forget. Having Alt tags can help screen readers and scrapers detect your content.  There are a few sites that will check to see if you're sit has all of your tags. One site I found: http://www.contentforest.com/seo-tools/image-alt-checker